### DISCUSSION TITLE
Module Federation consultation session

### DISCUSSION THREAD

**Discussion by danleydmello** [module-federation/module-federation-examples#2244] [General]
Hi Zack,

We are starting a new project on composing microfrontends and would like to use module federation.
I have experimented with module federation to a certain extent but have a few questions and unknowns that I will like to discuss with you.

One hour group session with you will be good opportunity for us

If you agree I will go ahead and book the session [here](https://calendly.com/scripted-alchemy/1-hr-group-consult?month=2022-08).
Is there a limit of participants in the group session? There will be about 4 to 5 people attending.

Looking forward to hearing back from you.

Regards
Danley

**Comment by danleydmello** [comment]
@ScriptedAlchemy , any comment on this?

**Reply by ScriptedAlchemy** [reply]
Yeah that's fine.

**Comment by danleydmello** [comment]
@ScriptedAlchemy , thats great , I'll book slot in first week of october Thanks
